examples/tv-casting-app/linux/main.cpp

```diff
@@ -216,8 +216,9 @@ void DeviceEventCallback(const DeviceLayer::ChipDeviceEvent * event, intptr_t ar
 {
     if (event->Type == DeviceLayer::DeviceEventType::kCommissioningComplete)
     {
-        chip::NodeId tvNodeId             = chip::DeviceLayer::DeviceControlServer::DeviceControlSvr().GetPeerNodeId();
-        chip::FabricIndex peerFabricIndex = chip::DeviceLayer::DeviceControlServer::DeviceControlSvr().GetFabricIndex();
+        chip::NodeId tvNodeId = chip::DeviceLayer::DeviceControlServer::DeviceControlSvr().GetPeerNodeId();
+        chip::FabricIndex peerFabricIndex =
+            chip::DeviceLayer::DeviceControlServer::DeviceControlSvr().GetFailSafeContext().GetFabricIndex();
```

Comment:
This doesn't make sense to me.  This is running async after CommissioningComplete, right?  The failsafe context has been torn down.  The fabric index there is stale data, and could in fact be a different fabric index by now.

Sounds like we should be putting the fabric index into the event itself, along with status.  Followup for that and for removing this broken dependency on the failsafe bits.

But also (followup issue) this code is not checking the status, so if a failsafe times out it will run and do ... something.  Probably with random values for the node id and fabric index....

We really need to stop having totally broken example code.

Reply:
https://github.com/project-chip/connectedhomeip/issues/15547